cpp/arcticdb/processing/operation_dispatch_binary.hpp

```diff
@@ -79,7 +79,7 @@ VariantData binary_membership(const ColumnWithStrings& column_with_strings, Valu
 
     util::BitSet output_bitset;
     constexpr auto sparse_missing_value_output = std::is_same_v<std::remove_reference_t<Func>, IsNotInOperator>;
-    details::visit_type(column_with_strings.column_->type().data_type(),[&] (auto col_tag) {
+    details::visit_type(column_with_strings.column_->type().data_type(),[&, sparse_missing_value_output] (auto col_tag) {
```

Comment:
Was this because of a compiler error? Shouldn't the `&` capture it?

Reply:
It was required because it is not captured by value by default unless explicitly specified (in the scope of the lambda functions, `sparse_missing_value_output` was not declared nor defined).